### DISCUSSION TITLE
Use undetected-chromedriver as a git submodule

### DISCUSSION THREAD

**Discussion by txtsd** [FlareSolverr/FlareSolverr#699] [Ideas]
Instead of committing all the files from the [undetected-chromedriver](https://github.com/ultrafunkamsterdam/undetected-chromedriver) repo as files here, why not add it as a git submodule?

**Comment by ngosang** [comment]
I patched several things in the module:
* Hidden window in Windows
* Close zombie processes

**Reply by txtsd** [reply]
Can those patches be submitted upstream?

**Reply by ngosang** [reply]
Some, but not all of them. I will upstream some before the next release.

**Reply by txtsd** [reply]
> Some, but not all of them. I will upstream some before the next release.

Great!

You could also maintain your patches on your own fork of undetected-chromedriver, and pull that in as a submodule.
